pyproject.toml

```diff
@@ -160,8 +160,6 @@ filterwarnings = [
     '''ignore:.*an autograd kernel was not registered to the Autograd key.*:UserWarning''',
     # Ignore save_state_dict / load_state_dict deprecation warnings
     '''ignore:'.*_state_dict' is deprecated and will be removed in future versions.*:UserWarning''',
-    # Ignore HF deprecation which affects their own libraries
```

Comment:
this was just old right? not doing anything anyways

Reply:
it wasn't actually working for some reason...but the warnings go away entirely on the newest transformers versions so irrelevant now